pkg/trace/api/profiles.go

```diff
@@ -87,15 +87,21 @@ func (r *HTTPReceiver) profileProxyHandler() http.Handler {
 		tags.WriteString("_dd.origin:lambda")
 	}
 
-	// Azure Container App metadata
-	if subscriptionID, ok := r.conf.GlobalTags["subscription_id"]; ok {
-		tags.WriteString(fmt.Sprintf(",subscription_id:%s", subscriptionID))
+	azureContainerAppTags := []string{
```

Comment:
If this slice isn't going to change, then I'd recommend placing it as a global.

Reply:
Done!

https://github.com/DataDog/datadog-agent/blob/01dfdba8ce556a78d59c1a344b4151631df8628a/pkg/trace/api/profiles.go#L33-L42